pkg/authorization/authorizer/scope/converter.go

```diff
@@ -178,37 +179,37 @@ func (userEvaluator) Describe(scope string) (string, string, error) {
 	}
 }
 
-func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter authorizationlister.ClusterPolicyLister) ([]authorizationapi.PolicyRule, error) {
+func (userEvaluator) ResolveRules(scope, namespace string, _ rbaclisters.ClusterRoleLister) ([]rbac.PolicyRule, error) {
 	switch scope {
 	case UserInfo:
-		return []authorizationapi.PolicyRule{
-			{Verbs: sets.NewString("get"), APIGroups: []string{userapi.GroupName, userapi.LegacyGroupName}, Resources: sets.NewString("users"), ResourceNames: sets.NewString("~")},
+		return []rbac.PolicyRule{
```

Comment:
Long lines make my eyes bleed.
Short lines are much easier to read and parse in my brain.